frontend/src/components/ExternalDb.vue

```diff
@@ -27,14 +27,24 @@
         </li>
       </ul>
     </div>
+    <div v-else>
```

Comment:
```suggestion
    <div v-else-if="errorMessage.length > 0">
```

The error notification appears to flash as the page is loading, this should make it not show by default.

Reply:
Thanks @e0 for pointing this out. It has been fixed by ab38b8ef with a similar if condition.

Reply:
Slightly less ideal with a potential empty div but acceptable.

Reply:
As shared in another screenshot, the message appears wider than our usual `container` width, so one idea could be to add it here:

```suggestion
    <div v-else class="container">
```